docs/installation.rst

```diff
@@ -0,0 +1,139 @@
+
+.. note::
+
+    Building the documentation is in general not necessary unless you are
+    writing new documentation or do not have internet access, because
+    the latest versions of hvpy's documentation should be available
+    at `hvpy.readthedocs.io <https://hvpy.readthedocs.io/>`_ .
+
+Dependencies
+^^^^^^^^^^^^
+
+Building the documentation requires the ``hvpy`` source code and some
+additional packages. The easiest way to build the documentation is to use `tox
+<https://tox.readthedocs.io/en/latest/>`_ as detailed in
+:ref:`hvpy-doc-building`.
+
+On the other hand, if you wish to call Sphinx manually to build the
+documentation, you will need to make sure that dependencies are
+installed. If you use conda, the easiest way to install the dependencies is
+with::
+
+    conda install -c conda-forge sphinx-book-theme sphinx-autodoc-typehints sphinx-automodapi
+
+Without conda, you install the dependencies by specifying ``[docs]`` when
+installing ``hvpy`` with pip::
+
+    pip install -e '.[docs]'
+
+.. Note::
+    The ``pip`` install methods above do not include `Graphviz
+    <http://www.graphviz.org>`_.  If you do not install this package separately
+    then the documentation build process will produce a very large number of
+    lengthy warnings (which can obscure bona fide warnings) and also not
+    generate inheritance graphs.
+
+.. _hvpy-doc-building:
+
+Building
+^^^^^^^^
+
+There are two ways to build the ``hvpy`` documentation. The easiest way is to
+execute the following tox command (from the ``hvpy`` source directory)::
+
+    tox -e build_docs
+
+If you do this, you do not need to install any of the documentation dependencies
+as this will be done automatically. The documentation will be built in the
+``docs/_build/html`` directory, and can be read by pointing a web browser to
+``docs/_build/html/index.html``.
+
+Alternatively, you can do::
+
+    cd docs
+    make html
+
+And the documentation will be generated in the same location. Note that
+this uses the installed version of hvpy, so if you want to make sure
+the current repository version is used, you will need to install it with
+e.g.::
+
+    pip install -e .[docs]
+
+before changing to the ``docs`` directory.
+
+In the second way, LaTeX documentation can be generated by using the command::
+
+    make latex
+
+The LaTeX file ``hvpy.tex`` will be created in the ``docs/_build/latex``
+directory.
+
+
+
+Reporting Issues/Requesting Features
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+As mentioned above, building the documentation depends on a number of Sphinx
+extensions and other packages. Since it is not always possible to know which
+package is causing issues or would need to have a new feature implemented, you
+can open an issue in the official `hvpy repository <https://github.com/Helioviewer-Project/python-api/issues>`__.
+However, if you wish, you can also open issues in the `helioviewer API repository <https://github.com/Helioviewer-Project/api/issues>`__.
```

Comment:
```suggestion
As mentioned above, building the documentation depends on a number of Sphinx extensions and other packages.
Since it is not always possible to know which package is causing issues or would need to have a new feature implemented, you can open an issue in the official `hvpy repository <https://github.com/Helioviewer-Project/python-api/issues>`__.
However, if you wish, you can also open issues in the `helioviewer API repository <https://github.com/Helioviewer-Project/api/issues>`__.
```

This doesn't belong at the bottom of the installation page nor do I understand the link between the documentation build and getting new features or reporting bugs.

Reply:
I think this bit might more ideal on the landing page.